stack/jenkins_VM.yml

```diff
@@ -0,0 +1,110 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
change the name of the file as jenkins_vm and not jenkins_VM
Capital letter is not needed for those files

Reply:
corrected remarks